src/components/layout/layout.tsx

```diff
@@ -31,6 +31,13 @@ const Main = styled.main`
   margin: 0 auto;
 `;
 
+const HeaderPositioningLayout = styled.div`
```

Comment:
I know that you suggested this name, @georgiee... I'm not sure about this `Layout` suffix, though. What's the reasoning? Why not the classical terms `Wrapper` or `Container`? Or `LayoutPositioning` instead of `PositioningLayout`? To me this is simply not a "Layout".

Reply:
It's a rough idea and I'm happy about new ideas. `HeaderContainer` would work for me. What about `HeaderStickyContainer`? wrapper & container are kind of overloaded and hides the nice idea of separating the sticky concern from the header itself.

Reply:
I very much like the idea of making it clear in the name that this is about the positioning. `HeaderStickyContainer` works well for me. To make it technically more correct it could also be `HeaderFixedContainer`. It won't make a lot of difference.

For now I suggest to go with `HeaderStickyContainer` 👍